tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.SurfaceArea.netstandard20.debug.bsl

```diff
@@ -7,6 +7,23 @@
 ! AssemblyReference: System.Reflection.Emit.ILGeneration
 ! AssemblyReference: System.Reflection.Metadata
 ! AssemblyReference: netstandard
+<>f__AnonymousType4056634344`3[<Dependencies>j__TPar,<Dependents>j__TPar,<Nodes>j__TPar]: <Dependencies>j__TPar Dependencies
```

Comment:
This doesn't look like the surface area to me. At least in terms of we wanted to stabilize the APIs that the FCS users would be using.

Reply:
Yeah I have no idea why this appeared in the baseline. Hopefully it can be removed.

Reply:
Yeah it seems it shouldn't be there: https://dev.azure.com/dnceng-public/public/_build/results?buildId=873483&view=ms.vss-test-web.build-test-results-tab&runId=22787518&resultId=105207&paneView=debug

No idea why it was generated on my machine.

Reply:
Oh I see it, it's the Debug helper on DependencyGraph. The whole DependencyGraph should just be `internal`.